tensorflow-framework/src/main/java/org/tensorflow/framework/data/Dataset.java

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.

```

Comment:
AFAIK, you don't work at Oracle :)

I suggest you change all copyrights to `Copyright 2020 The TensorFlow Authors. All Rights Reserved.`, like most of the files in the core module, that is what I'm using too.

Reply:
Thanks for catching! I'll make that change.